### ISSUE TITLE
Would love some help!

### ISSUE THREAD

**Issue by SneakySphynx** [Maxx53/SCMBot#427]
Im using: 

Requests Limit: 5
Delay Between Requests: 100
Item Delay: 2000

Scanning for - 5 Items 

Been trying for like 3 weeks, with 0 buys.
What the hell am i doing wrong? :D
Got 31 Euro that i put the max price on, even added ak redline at 1 euro below max price to try.
And have not bought anything.

Any tips on what i should bot to make small profits or whatever, i dont mind aslong as i make a little.
I mean 3 weeks? come on :p 

Really thankfull for all the help i can get.

Im sitting on Fiber Optic internet, 100mb / 100mb 
Only user, and 0 downtime. If that helps.

**Comment by mtzi** [comment]
try testing with 0.03$ item to see if bot can buy, or if u have cookies problem, also do the scans happen normal? like ever 2-3 sec? maybe sometimes it lags and scans once every 10 sec?

**Comment by SneakySphynx** [comment]
![](http://i.imgur.com/sRI62ms.png)

Thats how my scan times look, i will try with a 0.03 item will post an update soon mate.
Thanks alot for getting back to me. :)

**Comment by SneakySphynx** [comment]
Okay so i added a Phoenix weapon case for 0.03 and put buy quantity to 1

Ended up with this:


![](http://i.imgur.com/mXL6xlm.png)

**Comment by SneakySphynx** [comment]
I login in using:

Sessionid
webtradeeligibility
steamlogin

Im not using:
Since i dont know where to find them.

steamMachineAuth
steamLoginSecure

**Comment by mtzi** [comment]
thats the problem, if u dont have these 2, u DONT buy, u get the "problem purchasing error" i had it too. try using firefox to find these cookies, i had same problem, and 1 more guy, we both found it on firefox. also steamMachineAuth has random numbers behind it. like steamMachineAuth79123 thats what u want.

**Comment by SneakySphynx** [comment]
Aaah, i will get right on that. Well that was a 3 week waste :p 

Thanks alot for helping out, got one final question you'll probably dont want to answer but it's worth a try!

Is there anything in particular i can scan for, that will give me many, but small profits?

Im currently going with 3-4 Hightier knives and 1x low ak redline. 

Thanks again! <3

**Comment by SneakySphynx** [comment]
By the way, should i search for Newly Listed, or Item Page?

**Comment by mtzi** [comment]
newly listed is better if item is popular, example new case, everybody opens, everybody sells.
for slower items like knives its better to item page, however difference is newly listed can look infinite items because it scans 1 page (the newly listed) when if u put too many item pages u get errors because of many requests.

**Comment by SneakySphynx** [comment]
God damnit, installed firefox logged in etc.
Fixed all cookies, and still getting errors.


![](http://i.imgur.com/V7nF803.png)


No idea why, tried with different delays since it was a case for 0.03. 

:/

**Comment by mtzi** [comment]
if u installed firefox right now, its still tradeblocked for a week, u need to find cookies from browser where u are not tradeblocked.

**Comment by SneakySphynx** [comment]
Ah, so can i use chrome cookies for everything except the:

steamMachineAuth
steamLoginSecure

Or that wont work maybe?

**Comment by mtzi** [comment]
also, my chrome just now showed the cookies u were missing, maybe you can try like 3-4 times to open steamcommunity and maybe they show. sometimes they dont show, sometimes they do. if u install new browser its tradeblocked for 1 week (new device thing)

**Comment by SneakySphynx** [comment]
Doesnt say anything when im browsing market about trade block, maybe beacuse i used steam authenticator?

**Comment by mtzi** [comment]
so on firefox u dont have tradeblock?

**Comment by mtzi** [comment]
u can use cookies from whatever browser u DONT have market block

**Comment by SneakySphynx** [comment]
Im going to try and buy a case from market on firefox hang on!

**Comment by SneakySphynx** [comment]
Yeah i can buy from firefox : )

**Comment by mtzi** [comment]
so the cookies should be correct, try using all 5 from firefox?

**Comment by SneakySphynx** [comment]
Yeah that's what im doing atm, wont work tho :s 
Will try to relog the scmbot, hang on :)

**Comment by SneakySphynx** [comment]
Redid everything, and still giving me errors. No idea wtf's going on. :/

**Comment by mtzi** [comment]
me neither man. sorry :/ ask @Maxx53 he might know more

**Comment by SneakySphynx** [comment]
I added the case to " item page tab "
And it gave me an error: Item is not supported in html

**Comment by SneakySphynx** [comment]
![](http://i.imgur.com/X5G91xn.png)

**Comment by SneakySphynx** [comment]
![](http://i.imgur.com/jU76Z2I.png)

**Comment by mtzi** [comment]
yes, items listed as commodities for steam market (keys cases etc) dont work as HTML, u need to go to option and scan JSON if u want cases, but if u scan JSON, u cant look for knives etc.

**Comment by mtzi** [comment]
you either scan HTML for items (not cases not keys), or u scan JSON for ONLY keys/cases no skins

**Comment by SneakySphynx** [comment]
Ah maybe i should turn of html, and try if it works to buy a case.
Maybe thats why it gives me an error. :)

**Comment by SneakySphynx** [comment]
![](http://i.imgur.com/EyH3H4j.png)

**Comment by mtzi** [comment]
ive never had this much trouble... i dont know whats happening mate.. this shouldnt be happening

**Comment by SneakySphynx** [comment]
My standard luck. :( thanks for trying mate, really appreciate it.

**Comment by mtzi** [comment]
just wait for max to come, he sure knows more. @Maxx53 help this guy

**Comment by Maxx53** [comment]
items with new buy order interface is not supported. such as http://steamcommunity.com/market/listings/730/Operation%20Phoenix%20Weapon%20Case

**Comment by SneakySphynx** [comment]
Aha, ill try another one. Thanks for the reply!

**Comment by SneakySphynx** [comment]
Are my scan settings looking good, if we count aside all my problems i've had?

Requests Limit: 5
Delay Between Requests: 100
Item Delay: 2000
HTML Search: Yes

Would that work for knives/skins? :)

**Comment by mtzi** [comment]
less item delay better searching BUT the limit is where it gives "not supported in html" error, for 5 items i remember using 1700 delay (which is faster than 2000), try to reduce it as much as u can BUT WHILE NOT GETTING errors, it looks good tho even without changing it

**Comment by SneakySphynx** [comment]
I added my 4 knives to item page tab, and im currently at 1200 delay without errors will lower it more.
And i got an ak 47 elite build on newly listed tab, just to try. :)

**Comment by mtzi** [comment]
all works then?

**Comment by SneakySphynx** [comment]
Looks like it, just waiting for it to buy a ak 47 elite build, will raise the wished price just to get it over with. Knives are scanning fine. Im down to 500 delay on all 4 still no errors. 

Should i crank it down even more? =)

**Comment by mtzi** [comment]
dont go much lower than 500. ppl say it might get u banned if lower than 500

**Comment by SneakySphynx** [comment]
Alright ill stay at 500-700 ish

**Comment by mtzi** [comment]
if u want try buying 1 shit skin, autosniper polar camo 0.04 cents

**Comment by mtzi** [comment]
so u can test the cookies etc

**Comment by SneakySphynx** [comment]
Yeah it worked, put it on " item page " and it bought it after 3 sec. :)

**Comment by SneakySphynx** [comment]
Also changed delays to 600 to be safe, well let the waiting begin! ^^ 

Thanks alot for all your help, it is extremely appreciated!

**Comment by SneakySphynx** [comment]
Also the final question haha, can i have like 4 knives on " item page tab "
While having a few weapon skins on " newly listed " like popular ones? 
Or maybe better to keep them on item page tab aswell? :)

**Comment by mtzi** [comment]
if u try to put all the weapons on item page ull start getting errors, especially if they are more than 10-12 in total, and you will have to make scan slower
if u have like 20 gun skins better keep in newly listed. and knives on item tab, however item tab is said to work better it has 100% chance to buy if they put it low (and someone else doesnt get it first), whereas in newly listed it might not pop at all.

**Comment by mtzi** [comment]
try to find a balance. put the better items in item tab 5-10 and leave the rest on newly listed if u have more

**Comment by SneakySphynx** [comment]
Ah okay, i was thinking about adding like 5 knives / 3-5 top skins at Item page & Random low/med skins like 5-10 at newly listed.

Should be okay? :)

**Comment by mtzi** [comment]
sounds good

**Comment by SneakySphynx** [comment]
@mtzi Are theese settings alright? 

![](http://i.imgur.com/OGoneML.png)

Getting scan times like this with 4 items on " Item Page " tab.


![](http://i.imgur.com/rwajLyR.png)

If i add more, the scan times increase to 6 second intervalls.
Missed 2 knives i've been scanning for, just wanna make sure my settings are optimal :)

**Comment by KeooR** [comment]
Hello, like @SneakySphynx i don't know if my settings are good and safe for buy one item many times.
Here are my settings : 
![laphoto](https://cloud.githubusercontent.com/assets/13817999/9569226/477cd016-4f65-11e5-8e6e-9ec4c107342d.PNG)
( a lot of people buy before me ) Sorry for my bad english, Thanks.

**Comment by mtzi** [comment]
@SneakySphynx also about the knives, if you DONT have 100mbs optic fiber net, you wont buy items like asiimov's and knives, i stick to 4-5$ items, and still i get like 10 "someone bought item before you" for each 1 i buy.

**Comment by KeooR** [comment]
@mtzi Haha :D first thanks for your fast reply, and my settings are they optimizable ?

**Comment by mtzi** [comment]
i have 0 delay between requests, not 100, even tho 0 is NOT so safe. threads are ok at 3, 5 might be useless. simultaneous requests should be 1-5, depending on settings, more than 5 is dangerous. i use 2 for looking at 5 items.
i am gonna open a new question for resell tho, something is weird for me

**Comment by KeooR** [comment]
And for looking at 1 item. what do you recommande ?

**Comment by KeooR** [comment]
And for delay here : ? ![capture](https://cloud.githubusercontent.com/assets/13817999/9569475/f34d9f9a-4f6c-11e5-8bcb-677bc1ccf5fe.PNG)
600 is good or 0 is better ?

**Comment by mtzi** [comment]
anything below 500 is bannable. stick to 600 its fine. if you wanna go faster u can reduce it, but lower number than 300 is RLY scary to get ban

**Comment by SneakySphynx** [comment]
@mtzi Heya mate! 
Sry for the slow response, been hella' busy :D

I'm actually sitting on a 100/100 mbit fiber optic line, only user aswell. Haha.
But i will change delay between requests to 0, and threads to 2.

Also how do you get that window, so you can see how many missed purchases etc? :O
Teach me senpai <3

**Comment by mtzi** [comment]
maxx told me in another topic, let me find it and link it, had to recompile bot, but its not a big deal, copy paste 1 line of text.

Maxx53 commented 24 days ago
@mtzi u can do it yourself brah
after this string
https://github.com/Maxx53/SCMBot/blob/master/scr/SCMBot/MainUtils.cs#L419
add

Main.AddtoLog(content);

and thats it
open the file from your PC to editor. add the text, compile with visual basic again, it now shows the error.

**Comment by SneakySphynx** [comment]
@mtzi Oh, nice one! Thanks mate.
Now im just experimenting, trying to find something to keep on buying/selling. Still havnt bought anything ^^

**Comment by mtzi** [comment]
it was like 3 months before my first buy too. and its tough because CSGO new case is coming out next monday, and prices are gonna be unstable again (like they were for ESL).
try something about 4$ thats not so popular like some mediocre m4 or ak (try to not do redline, thats way too botted)

**Comment by SneakySphynx** [comment]
@mtzi Yeah i've been checking around for a low item, but from the statistics it doesnt show a single missprice on them :s Will keep on looking :)

**Comment by KeooR** [comment]
Ok thank you very much :+1:   So with these settings I guess I 'm good :)
![capture](https://cloud.githubusercontent.com/assets/13817999/9577573/a013bc08-4fe2-11e5-962c-3c8ed86de76b.PNG)

**Comment by mtzi** [comment]
@SneakySphynx when 1 missprice is sold and 7 normal, it shows only the normal for that 1 hour. missprice is shown only in very rare items (knives).

@KeooR looks good. simultaneous request limit to 2 should work better if you have more than 2 items

**Comment by KeooR** [comment]
@mtzi Ok thanks ! I just change Number of threads to 3.

**Comment by SneakySphynx** [comment]
@mtzi Aaah, that explains alot!

**Comment by KeooR** [comment]
I do not understand very well what it means the line : " Number of threads ( Newly Listed ) " can you explain me please :3

**Comment by mtzi** [comment]
If you ever try to search newly listed you'll see it scans many times before waiting for the "item delay".
Number of threads is number of scans before it waits the item delay.
example, if threads = 1 -> scan 1 time, wait item delay, scan 1 time etc
if threads = 5 -> scan 5 times, wait item delay, scan 5 times etc
i have 3 with 1500 item delay when i scan newly listed, so i scan 3 times every 1500ms

**Comment by KeooR** [comment]
OK thanks, and 1500ms delay for newly listed is not to slow ? I have tested these tab ( newly listed ) for a popular item while one hour, no succes :/

**Comment by KeooR** [comment]
And this is normal ?
![why](https://cloud.githubusercontent.com/assets/13817999/9606677/20c1c59c-50c5-11e5-825d-02e1dfa7c638.png)

**Comment by mtzi** [comment]
@KeooR was at work, replied on other thread, chack that must be cos of many items, bot usually sends max 2 or 3 requests at same time, and if u have many items it repeats 2-3 every sec without going on next, if u increase delay to 1000 to have it be able to fully cycle through all the items on list might work better

**Comment by KeooR** [comment]
No, i have only one item :/

**Comment by mtzi** [comment]
the problem must be the servers, the bot gives lots of "fake ping" on these things. try to find from the ping sites the server that is the best for your country (go google, search ping from different locations, do the ping, find some server that has like 5-10 ping to you), then use go to the hosts of program and use that server. example coming 
23.223.72.17
23.223.72.18
23.223.72.19
23.223.72.20
23.223.72.21
23.223.72.22
23.223.72.23
23.223.72.24
23.223.72.25
23.223.72.26
23.223.72.27
23.223.72.28

This is how it should look like, find 1 host, then add the next 10 manually so it looks like this

**Comment by KeooR** [comment]
Ok thanks, see my screenshot.
![capture](https://cloud.githubusercontent.com/assets/13817999/9631399/ecfc0116-5180-11e5-90d0-19aab84e5a5e.PNG)

**Comment by mtzi** [comment]
@KeooR is it good for you? its every 1-2 sec now. (with 600 delay i had the same as u, 1 request per 1,3 seconds, the 100 requests between every 2 logs were 130 sec, i mean the log reset and started new every 130sec), i think its the fastest it goes :P

**Comment by KeooR** [comment]
@mtzi Yes is very good now. Thank you lot :+1:

**Comment by exoittream** [comment]
I need logs
